tools/clang/test/CodeGenSPIRV/rayquery_init_hs.hlsl

```diff
@@ -56,19 +56,20 @@ void doInitialize(RayQuery<RAY_FLAG_FORCE_OPAQUE|RAY_FLAG_SKIP_TRIANGLES> query,
     query.TraceRayInline(AccelerationStructure,RAY_FLAG_FORCE_NON_OPAQUE,0xFF,ray);
 }
 
+[outputtopology("triangle_ccw")]
 [outputcontrolpoints(MAX_POINTS)]
 [patchconstantfunc("mainConstant")]
 CONTROL_POINT main(InputPatch<VS_CONTROL_POINT_OUTPUT, MAX_POINTS> ip, uint cpid : SV_OutputControlPointID) {
 // CHECK:  %rayQueryKHR = OpTypeRayQueryKHR
 // CHECK:  %_ptr_Function_rayQueryKHR = OpTypePointer Function %rayQueryKHR
-// CHECK:  [[rayquery:%\d+]] = OpVariable %_ptr_Function_rayQueryKHR Function
+// CHECK:  [[rayquery:%[0-9]+]] = OpVariable %_ptr_Function_rayQueryKHR Function
     RayQuery<RAY_FLAG_FORCE_OPAQUE|RAY_FLAG_SKIP_TRIANGLES> q;
     RayDesc ray = MakeRayDesc();
-// CHECK:  [[accel:%\d+]] = OpLoad %accelerationStructureNV %AccelerationStructure
-// CHECK:  OpRayQueryInitializeKHR [[rayquery]] [[accel]] %uint_517 %uint_255 {{%\d+}} %float_0 {{%\d+}} %float_9999
+// CHECK:  [[accel:%[0-9]+]] = OpLoad %accelerationStructureNV %AccelerationStructure
+// CHECK:  OpRayQueryInitializeKHR [[rayquery]] [[accel]] %uint_257 %uint_255 {{%[0-9]+}} %float_0 {{%[0-9]+}} %float_9999
```

Comment:
Do you know why the ray flags changed from 517 to 257?

Reply:
No, only that 257 looks correct to me:

```
RAY_FLAG_FORCE_OPAQUE|RAY_FLAG_SKIP_TRIANGLES
= 0x01 | 0x100
= 0x101
= 257
```

These tests have never been run, so I assume the original test writer used different flags when they initially wrote the test then changed them without updating the CHECKs.